test/models/category_test.rb

```diff
@@ -1,7 +1,44 @@
 require "test_helper"
 
 class CategoryTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
+    site = create(:site)
+    @categories = create_list(:category, 5, site: site)
```

Comment:
Can you specify each category to individual name?
Currently `Category#name` depends on data initialization.